config/routes.rb

```diff
@@ -1,6 +1,5 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  root 'users#index'
+  get 'users/:id' => 'users#show', as: :user
+  get 'posts/:id' => 'posts#show', as: :post
```

Comment:
- [ ] Generally, you ought to create four restful _**routes**_ that `GET` resources from the application:
  - One for the `Users#Index` page
  - One for the `Users#Show` page
  - One for the `Posts#Index` page and 
  - One for the `Posts#Show` page

In the schema diagram, remember that the _**Post**_ belongs to the **_User_**. This means when you are creating the routes for the Posts, there should be a way to connect the users to their posts. Kindly implement this change. 👍 😎

 
#### Hint: You can read this [article](https://guides.rubyonrails.org/routing.html#:~:text=Nested%20routes%20allow%20you%20to%20capture%20this%20relationship%20in%20your%20routing.%20In%20this%20case%2C%20you%20could%20include%20this%20route%20declaration%3A) to learn how to create **_restful resourceful nested routes_**. Remember that you ought to create only the routes that are needed. For this, you can follow the pattern used in this [article](https://guides.rubyonrails.org/routing.html#:~:text=2.7.2%20Shallow%20Nesting). 💻 🖱️

Reply:
Thanks for the article man, you are a good reviewer!